app/views/govuk_publishing_components/components/docs/cards.yml

```diff
@@ -0,0 +1,143 @@
+name: Cards (experimental)
+description: A grid of cards that have a link and a description
+body: |
+  The component renders the cards as an unordered list element and visually presents it as a grid. It doesn't use the [GOV.UK Design System grid](https://design-system.service.gov.uk/styles/layout/#grid-system) but sets its own so that it can separately target desktop, tablet and mobile columns.
+
+  You can display the grid in three or two columns on desktop.
+
+  You must set a `href` for all the links.
+
+  This component is currently experimental. If you are using it, please feed back any research findings to the GOV.UK Explore team.
+accessibility_criteria: |
+  The component must:
+
+  -  use the correct heading level hierarchy eg. if the component heading uses a`<h2>`, the subheadings must use a `<h3>`
+
+  The component should:
+
+  -  set the component heading to tell users what the list contains
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      heading: Topics
```

Comment:
This relates to the open comment on https://github.com/alphagov/govuk_publishing_components/pull/2624#discussion_r811984893.

Reply:
My impression was that we favour an accessible example over a minimal one – this mental model made us [update the default radio example](https://github.com/alphagov/govuk_publishing_components/pull/2485). Happy to discuss this within the community though.

Reply:
There's a separate thread about the need for the heading. I'd hope that every example was accessible though...

Reply:
Since there is precedence in this repo for including a heading or other best practise elements in the first example I'm going to resolve this conversation, similar to https://github.com/alphagov/govuk_publishing_components/pull/2624#discussion_r816626444. The component is still experimental and we can always revisit this if we get feedback to the contrary for what we do here.